api/controllers/inquiry-controller.ts

```diff
@@ -48,15 +48,17 @@ export const updateInquiry = async (req: Request, res: Response) => {
  * @author dallascrichmond
  */
 export const deleteInquiry = async (req: Request, res: Response) => {
+    const { id } = req.params;
+    const inquiryID: number = +id;
```

Comment:
What happens here if id is undefined?

Reply:
Good point! I'll add a test for that/ make alterations if needed!

Reply:
@dbarkowsky I added two new tests for invalid IDs when deleting and updating inquiries